server/api/test.get.ts

```diff
@@ -0,0 +1,21 @@
+// this is a only GET request handler for /api/test
+// ! only a example, you can delete this file
+import DBManager from "../DBManager/DBManager";
+
+export default eventHandler(async (event) => {
+    // ! replace the config in the init with the useRunTimeConfig hook, which uses the runtimeConfig from nuxt.config.ts
+    const db = DBManager.instance ? DBManager.instance : await DBManager.init({
+        prod: false,
+        DB_URL: "mongodb://localhost:27017",
+        DB_NAME: "test",
+    });
```

Comment:
falls das nur ein Beispiel code ist, dann ok.

wenn das aber das Use-pattern sein soll, dann ist das nicht gut, weil man will ja gerade nicht jedes mal (und überall im code) die init parameter angeben.

d.h. das init muss man einer(1) zentralen stelle sein.

Reply:
man könnte auch in der Klasse selbst die `useRuntimeConfig()` aufrufen. Sollte funktionieren